### ISSUE TITLE
email create not working with ESM

### ISSUE THREAD

**Issue by arjunyel** [shellscape/jsx-email#221]
- Component or Package Name: jsx-email
- Component or Package Version: 2.0.8
- `@jsxp-email/cli` Version?: 2.0.8
- Operating System (or Browser):  macOS Sequoia 15.0.1
- Node Version: 20.18.0
- Link to reproduction: https://github.com/arjunyel/jsx-email-esm-issue

### Expected Behavior

`npx email create BatmanEmail --out=./emails` works

### Actual Behavior

```bash
ReferenceError: __dirname is not defined
    at command (file:///Users/test/test/email/node_modules/jsx-email/dist/esm/cli/commands/create.mjs:33:42)
```

### Additional Information

`patch-package` fix:

```patch
diff --git a/node_modules/jsx-email/dist/esm/cli/commands/create.mjs b/node_modules/jsx-email/dist/esm/cli/commands/create.mjs
index 503140a..43bf699 100644
--- a/node_modules/jsx-email/dist/esm/cli/commands/create.mjs
+++ b/node_modules/jsx-email/dist/esm/cli/commands/create.mjs
@@ -3,6 +3,10 @@ import { join, resolve } from 'path';
 import chalk from 'chalk';
 import mustache from 'mustache';
 import { parse as assert, boolean, object, optional, string } from 'valibot';
+import { dirname } from 'node:path';
+import { fileURLToPath } from 'node:url';
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
 const { log } = console;
 const CreateOptionsStruct = object({
     jsx: optional(boolean()),
@@ -30,7 +34,7 @@ export const command = async (argv, input) => {
     assert(CreateOptionsStruct, argv);
     const [name] = input;
     const { jsx, out } = argv;
-    const template = await readFile(join(__dirname, '../templates/email.mustache'), 'utf8');
+    const template = await readFile(join(__dirname, '../../../templates/email.mustache'), 'utf8');
     const data = {
         name,
         propsType: jsx ? '' : ': TemplateProps',
```

**Comment by shellscape** [comment]
Thanks for the issue. Will take a look at this for you tonight.

**Comment by shellscape** [comment]
Fixed in #222 v2.0.9
